optimade/server/mappers/entries.py

```diff
@@ -179,7 +173,7 @@ def all_length_aliases(cls) -> Iterable[Tuple[str, str]]:
         from optimade.server.config import CONFIG
 
         return cls.LENGTH_ALIASES + tuple(
-            CONFIG.length_aliases.get(cls.ENDPOINT, {}).items()
+            CONFIG.length_aliases.get(cls.ENDPOINT, {}).items()  # type: ignore[call-overload]
```

Comment:
```suggestion
            CONFIG.length_aliases.get(cls.ENDPOINT, {}).items()
```
For me, this "ignore" is also unnecessary.